mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping

```diff
@@ -94,12 +94,14 @@ CLASS net/minecraft/class_350 net/minecraft/client/gui/widget/EntryListWidget
 	CLASS class_351 Entry
 		FIELD field_22752 list Lnet/minecraft/class_350;
 		METHOD method_25343 render (Lnet/minecraft/class_4587;IIIIIIIZF)V
-			ARG 2 y
-			ARG 3 x
+			ARG 1 matrices
+			ARG 2 x
+			ARG 3 y
 			ARG 4 width
 			ARG 5 height
 			ARG 6 mouseX
 			ARG 7 mouseY
+			ARG 10 delta
```

Comment:
`tickDelta`?

Reply:
I've seen both thrown around, would be nice if we standardized on a name

Reply:
If it is a fraction between 0.0 and 1.0 for lerping a value between the last tick and present tick value, then it's `tickDelta`. If it is not tick related then just `delta`